README.md

```diff
@@ -9,18 +9,18 @@
 ## Features
```

Comment:
typo fixes are really welcome, but why changing SQL from lower to UPPER? I believe that is not necessary, plus the rendered SQL code is generally lower, so let's keep lower and changes to the minimum please.

Reply:
There was inconsistency between upper and lower case so I just picked upper as the standard. I'll change to lower case.

Reply:
@nicor88 I've changed SQL to use lower case.

Reply:
Thanks, that's a classic dilemma isn't it? - I'm sticking to lower nowdays, as I'm some less keycap to press :D

Reply:
I also get frustrated dealing with that caps lock key but it's much easier to read SQL, especially more complex scripts, when capitalizing keywords.